logs/logs.go

```diff
@@ -0,0 +1,53 @@
+// Package logs provides the standard interface and handler for OpenFaaS providers to expose function logs.
+//
+package logs
+
+import (
+	"fmt"
+	"time"
+)
+
+// Request is the query to return the function logs.
+type Request struct {
+	// Name is the function name and is required
+	Name string `json:"name"`
+	// Instance is the optional container name, that allows you to request logs from a specific function instance
+	Instance string `json:"instance"`
```

Comment:
Is this inspired/copied from anywhere in terms of naming / structure?

Reply:
OK so Name -> FunctionName?

Instance -> the specific replica the function? i.e. get me the logs from replica 3/10?

What would be a valid identifier for "Instance" on Swarm vs Kubernetes?

Reply:
In swarm `Instance` == `Task.Name`
In k8s, `Instance` == `Pod.Name`